common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -495,4 +502,15 @@ public static int getCachedResultCount() {
         return sCommandResultCache.getSize();
     }
 
+    private static void setCorrelationIdOnResult(@NonNull final CommandResult commandResult,
+                                                 @NonNull final String correlationId) {
+        // set correlation id on Local Authentication Result
+        if (commandResult.getResult() != null &&
```

Comment:
I think it would be a good idea to comment on why we have the check for localAuthenticationResult.  I'm assuming for the broker scenarios, but adding the comment will help future readers.

Reply:
Sure, I'll add that - the check is there because `CommandResult` is a generic object and can contain the result of other commands such as GetAccount command or GetDeviceCommand and those results are not instances of `LocalAuthenticationResult` and don't contain a property for correlation id.

Reply:
I think it's the local prefix that confuses me here.  So perhaps the comment isn't required...